mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -1664,14 +1665,37 @@ impl MarketMakerIt {
         .map_err(|e| ERRL!("{:?}", e))
     }
 
-    /// Currently, we cannot wait for the `Completed IAmrelay handling for peer` log entry on WASM node,
-    /// because the P2P module logs to a global logger and doesn't log to the dashboard.
+    /// Check if the node is connected to at least one seednode,
+    /// the rpc is used instead of checking the log for DEBUG messages (to opt out p2p debug logging)
     #[cfg(not(target_arch = "wasm32"))]
     pub async fn check_seednodes(&mut self) -> Result<(), String> {
-        // wait for at least 1 node to be added to relay mesh
-        self.wait_for_log(22., |log| log.contains("Completed IAmrelay handling for peer"))
+        let timeout_sec = 22.;
+        let start = now_float();
+        loop {
+            let res = self.rpc(&json!({
+                "userpass": self.userpass,
+                "method": "get_directly_connected_peers",
+            }))
```

Comment:
i guess that's a different check than what the funciton name tells, ha?

this queries connected peers, but in this function we wanna check connected seeds.
not sure if the old implementation used to check for seeds and not any peer (the log we used to wait for doesn't really point that it was waiting specifically for a seed).

it doesn't matter i think since in all test scenarios we have a seed peer connected to non-seed peer. we might just wanna rename the function to reflect the job it's doing.

Reply:
renamed,
tyvm for the notes